src/Microsoft.PowerShell.Commands.Utility/commands/utility/AddType.cs

```diff
@@ -1106,31 +1023,10 @@ private void WriteTypes(Assembly assembly)
         private void CompileSourceToAssembly(string source)
         {
             CSharpParseOptions parseOptions;
-            if (IsCSharp(Language))
+            if (Language == Language.CSharp)
             {
                 switch (Language)
```

Comment:
No need to switch on `Language` anymore, since you already checked `Language == Language.CSharp`

Reply:
The same. If you are going to push more commits to the same PR, then ignore this comment for now. Otherwise, this comment needs to be resolved for this PR to be merged.

Reply:
`switch` removed.

I want keep the PR small. No plans add new code.